src/views/Home/RecentActivity/index.tsx

```diff
@@ -31,7 +31,6 @@ function RecentActivities() {
         url: 'server://projects/recent-activities/',
         method: 'GET',
         autoTrigger: true,
-        shouldPoll: () => 5000,
```

Comment:
Any reason why polling was removed?

Reply:
We decided to remove it for now. I did mess up the branches though. But, this is okay.